### DISCUSSION TITLE
Undefined index password in the web server logfile

### DISCUSSION THREAD

**Discussion by kanbeq** [datenstrom/community#710] [Report a bug] [answered]
`[DATE TIME] PHP Notice:  Undefined index: password in /home/xrlcvxyd/public_html/system/extensions/private.php on line 23`

I have about a thousand lines with this notice. Could it be done so that this notification is not generated?

**Comment by schulle4u** [comment, marked as answer]
Thanks for the bug report. There was a left over from the old API which caused this exhaustive logging behaviour. Please [download and install the latest private extension](https://github.com/schulle4u/yellow-extensions-schulle4u/tree/master/private), this should fix the problem.

**Reply by kanbeq** [reply]
It works. Thanks for the quick response
